setup.py

```diff
@@ -1,17 +1,16 @@
-import re
 import setuptools
-import sys
 
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
 setuptools.setup(
     name="honeybee-radiance-folder",
-    use_scm_version = True,
+    use_scm_version=True,
     setup_requires=['setuptools_scm'],
     author="Ladybug Tools",
     author_email="info@ladybug.tools",
-    description="Honeybee Radiance folder is a Python library to read, write and validate Radiance folder structure.",
+    description="Honeybee Radiance folder is a Python library to read, write and "
+    "validate Radiance folder structure.",
```

Comment:
You are missing `include_package_data=True` and (I think) the `MAIFEST.IN` file to ensure that the `.cfg` file is included in the package.

Reply:
Yes. You are right!